### ISSUE TITLE
make genre page cards, use dummy data for now

### ISSUE THREAD

**Issue by Aadarsh805** [Aman-Mandal/groovy#136]
![image](https://user-images.githubusercontent.com/95094057/195855624-dc241694-82a2-4407-96a8-a047df29adbd.png)

**Comment by DarkSoul26** [comment]
@Aadarsh805 can I contribute to this issue?

**Comment by sumit-022** [comment]
I can work on this. Please assign me this issue

**Comment by Aadarsh805** [comment]
> @Aadarsh805 can I contribute to this issue?

yes sir, go ahead assigned you

**Comment by Aadarsh805** [comment]
> I can work on this. Please assign me this issue

You can work on the navlist of genre page, there is another issue on that, I'll assign you on that if you wanna work on it, just drop a comment on that issue

**Comment by DarkSoul26** [comment]
Hey @Aadarsh805 can you tell whether browse page already exists?

**Comment by Aadarsh805** [comment]
> Hey @Aadarsh805 can you tell whether browse page already exists?

yes we have a genres page already but we are updating it to the new UI

**Comment by DarkSoul26** [comment]
Hey @Aadarsh805 can you please tell me where the Routes are present?

**Comment by MunavvarSinan** [comment]
can u assign me this task if it is not done yet
